test/image/test_image_info.py

```diff
@@ -29,25 +29,41 @@ def break_down_filename(image):
 
 
 def test_get_img_info():
-    short_keys = {"f", "h", "bpc", "w", "cs", "trns", "dp", "pal"}
+    short_keys = {"f", "h", "bpc", "w", "cs", "trns", "dp", "pal", "smask"}
     expected = json.loads((HERE / "image_info.json").read_text(encoding="utf8"))
 
     for path in sorted((HERE / "png_test_suite").glob("*.png")):
         if not path.stem.startswith("x"):
             blob = BytesIO(path.read_bytes())
             info = fpdf.image_parsing.get_img_info(blob)
-            short_info = {k: v for k, v in info.items() if k in short_keys}
+            short_info = {}
+            for k, v in info.items():
+                if k in short_keys:
+                    short_info[k] = v.decode("latin-1") if isinstance(v, bytes) else v
```

Comment:
Why is this call to `.decode()` needed?

Reply:
Because now `pal` is a sequence of bytes and when I tried to serialize it in JSON it threw `TypeError: Object of type 'bytes' is not JSON serializable` so before serialize it, I called `.decode()`

I also added `smask` in the `short_info`, it was not necessary, but I thought that could be tested too

Reply:
OK, alright!